about.html

```diff
@@ -18,6 +18,11 @@
     </section>
     <section class="nav-menu">
         <nav class="navbar">
+            <div class="hamburger">
```

Comment:
- [ ] Generally, on your `About Me` page, kindly work on the responsiveness, positioning, and center alignment as indicated in the picture below:

![image](https://github.com/ishimwepati/Conference-Page/assets/99715640/e60ecee6-b715-490e-9f4f-2ca0018f89c7)

Reply:
Done